app/views/queue_items/index.html.haml

```diff
@@ -18,7 +18,7 @@
               - @queue_items.each do |queue_item|
                 %tr
                   = hidden_field_tag "queue_items[][id]", queue_item.id
-                  %td= text_field_tag "queue_items[][position]", queue_item.position
+                  %td= text_field_tag "queue_items[][position]", queue_item.position, data: {video_id: queue_item.video.id}
```

Comment:
I can not see you would be using this update `data` in your feature spec. Why did you add it then?